packages/bsky/tests/__snapshots__/feed-generation.test.ts.snap

```diff
@@ -552,96 +552,6 @@ Array [
       "uri": "record(9)",
       "viewer": Object {},
     },
-    "reply": Object {
-      "parent": Object {
-        "$type": "app.bsky.feed.defs#postView",
```

Comment:
Any idea why these snaps changed?

Reply:
idk but these seem to fail intermittently on my local on main. I tried to debug for a bit and found out that somehow `invalidReplyRoot` comes out to be true for some replies which causes the tests to fail. however, it's not reliably reproduced. so I ended up disabling all failing tests and enabling 1 at a time and somehow ended up getting all green.